AppController/djinn.rb

```diff
@@ -4253,14 +4250,17 @@ def initialize_server()
 
     if not Nginx.is_running?
       Nginx.initialize_config()
-      Nginx.create_app_load_balancer_config(my_node.public_ip,
-        my_node.private_ip, AppDashboard::PROXY_PORT)
       Nginx.start()
       Djinn.log_info("Nginx configured and started.")
     else
       Djinn.log_info("Nginx already configured and running.")
     end
 
+    # TODO: As per trusty's version of haproxy, we need to have a
```

Comment:
Good point: I think it is a 'TODO: check newer versions when they are available'. On my system haproxy can start without any need to have a listenig socket.